docs/wiki/ModulesDesign.md

```diff
@@ -537,3 +538,45 @@ Parameter files in CARML leverage the common `deploymentParameters.json` schema
 - Likewise, the `name` parameter we have in most modules should give some indication of the file it was deployed with. For example, a `min.parameters.json` parameter file for the virtual network module may have a `name` property with the value `sxx-az-vnet-min-001` where `min` relates to the prefix of the parameter file itself.
 - A module should have as many parameter files as it needs to evaluate all parts of the module's functionality.
 - Sensitive data should not be stored inside the parameter file but rather be injected by the use of [tokens](./ParameterFileTokens.md) or via a [key vault reference](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/key-vault-parameter?tabs=azure-cli#reference-secrets-with-static-id).
+
+# Resource Removal
```

Comment:
@simonbMS , do you think it makes sense to move it to the [removal section](https://github.com/Azure/ResourceModules/wiki/PipelinesDesign#removal) of the pipeline design page?

There we have currently a brief decription we could extend with this content. I somewhat feel the removal is more pipeline/platform rather than Module-Design specific (as that focuses more on how a module should look like)

Reply:
Honestly I don't think it fits perfectly in neither place.
At the moment I moved it in the pipeline section, as the removal step was already mentioned there.

However, I think we should brainstorm a bit (outside of this PR 😄) on how to restructure the entire Wiki (this could even be one of the "outcome" of the hackathon).
Personally I would prefer a "user story based" structure (use/contribute), even if it could end up duplicating some content

Reply:
Sounds good to me